pypeit/wavecalib.py

```diff
@@ -475,6 +475,7 @@ def build_wv_calib(self, arccen, method, skip_QA=False):
                                              self.binspectral,
                                              nonlinear_counts=self.nonlinear_counts,
                                              nsnippet=self.par['nsnippet'])
+                                             #debug=True, debug_reid=True, debug_xcorr=True)
```

Comment:
Remove?

Reply:
I prefer to keep it.  I always have to hunt and peck to figure these
out when debugging.  Nice to have em here already.